docs/openid.md

```diff
@@ -446,3 +446,51 @@ The synapse config will look like this:
       config:
         email_template: "{{ user.email }}"
 ```
+
+## Django OAuth Toolkit
+
+[django-oauth-toolkit](https://github.com/jazzband/django-oauth-toolkit) is a
+Django application providing out of the box all the endpoints, data and logic
+needed to add OAuth2 capabilities to your Django projects. It supports
+[OpenID Connect too](https://django-oauth-toolkit.readthedocs.io/en/latest/oidc.html).
+
+**Configuration on Django's side:**
+
+1. Add an application: https://example.com/admin/oauth2_provider/application/add/ and choose parameters like this:
+  * `Redirect uris`: https://synapse.example.com/_synapse/client/oidc/callback
+  * `Client type`: `Confidential`
+  * `Authorization grant type`: `Authorization code`
+  * `Algorithm`: `HMAC with SHA-2 256`
```

Comment:
this needs more indentation to avoid starting a new list:
```suggestion
   * `Redirect uris`: https://synapse.example.com/_synapse/client/oidc/callback
   * `Client type`: `Confidential`
   * `Authorization grant type`: `Authorization code`
   * `Algorithm`: `HMAC with SHA-2 256`
```